### PR TITLE
v1.1.0 – Fixes to use of `!default` and re-organising optional components

### PR DESCRIPTION
**Pull request by ashleynolan**
### Fixed
- Issue with `!default` causing spacing and breakpoint maps to not be set correctly to not be set correctly (as already set previously, so not overriding properly).
- CHANGELOG typo fix

### Changed
- Moved optionally included components (via mixin) into `/optional` subfolder within `/components` directory
- Minor package updates

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,19 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
+v1.1.0
+------------------------------
+*October 3, 2018*
+
+### Fixed
+- Issue with `!default` causing spacing and breakpoint maps to not be set correctly to not be set correctly (as already set previously, so not overriding properly).
```
Comment (kevinrodrigues) [inline CHANGELOG.md]: `to not be set correctly to not be set correctly`

Comment (DamianMullins) [reply CHANGELOG.md]: To be sure, to be sure 🍀

Comment (ashleynolan) [reply CHANGELOG.md]: 🤦‍♂️

**Comment by ashleynolan** [comment]
@DamianMullins @kevinrodrigues Any chance of a re-review when you get 5 – have updated the `CHANGELOG`

**Pull request merged**